.github/workflows/release-winget.yaml

```diff
@@ -0,0 +1,38 @@
+name: "release-winget"
+on:
+  release:
+    types: [released]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - id: update-winget
+      name: Update winget repository
+      uses: mjcheetham/update-winget@v1.1
```

Comment:
Did I mis-remember that this was `v1.2` in your GCM Core one?

Reply:
[Yes, you used `v1.2`](https://github.com/microsoft/Git-Credential-Manager-Core/pull/332/files#diff-b4ee64863401eef676b49220519b9c6b6532f995c8b256ed6273b71237438fcaR12)

Reply:
Oh yikes, I can't believe I forgot to do that. Totally my bad.